pkg/chartutil/values_test.go

```diff
@@ -447,10 +447,12 @@ chapter:
 	if _, err := d.PathValue("chapter.doesntexist.one"); err == nil {
 		t.Errorf("Non-existent key in middle of path should return error: %s\n%v", err, d)
 	}
+	if _, err := d.PathValue(""); err == nil {
+		t.Errorf("Empty path should return error: %s\n%v", err, d)
```

Comment:
Should I even include the document in the complaint? Seems kind of irrelevant in this case.